esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -415,7 +416,18 @@ define([
         _onAboutClose: function (evt) {
             this.aboutDialog.hide();
         },
-
+		
+        _onLogout: function (evt) {
+            xhr("esp/logout", {
+                method: "post"
+            }).then(function(data){
```

Comment:
We are expecting that `data` contains the header information which means it has succesfully removed session information. @wangkx is there something more valuable that we can include here? Perhaps the 'You have logged out' text or something of the nature?